must-gather/scripts/bugzilla_attach.py

```diff
@@ -0,0 +1,406 @@
+#!/bin/python3
```

Comment:
please update references to kubevirt

Reply:
removed that script, was also copied from kubevirt must-gather, but is apparently not needed.